pom.xml

```diff
@@ -43,17 +43,17 @@
         <maven.version.range>[3.0.4,)</maven.version.range>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
         <project.build.locale>zh_CN</project.build.locale>
-
-        <springframework.version>[3.1.0.RELEASE,5.0.0.M1)</springframework.version>
-
+        
+        <springframework.version>[3.1.0.RELEASE,5.2.7.RELEASE]</springframework.version>
```

Comment:
If spring framework upgrade to new version, we still need to modify here,
can we keep the upper boundary open?

Reply:
I get it, how do you think '<springframework.version>[3.1.0.RELEASE,)</springframework.version>' ?

Reply:
Great